pkg/export/otel/metrics.go

```diff
@@ -229,14 +231,17 @@ type MetricsReporter struct {
 	attrGPUMemoryAllocations   []attributes.Field[*request.Span, attribute.KeyValue]
 	userAttribSelection        attributes.Selection
 	input                      <-chan []request.Span
+	processEvents              <-chan exec.ProcessEvent
 }
 
 // Metrics is a set of metrics associated to a given OTEL MeterProvider.
 // There is a Metrics instance for each service/process instrumented by Beyla.
 type Metrics struct {
-	ctx      context.Context
-	service  *svc.Attrs
-	provider *metric.MeterProvider
+	ctx                      context.Context
+	service                  *svc.Attrs
+	provider                 *metric.MeterProvider
+	resourceAttributes       []attribute.KeyValue
```

Comment:
I must remember the original resource attributes when the service is created for clean-up purposes. When a service is removed the attributes might be different if we had lost the K8S data.